04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,31 @@
-#write your code here
+def translate(str)
+    vowels = %w[a e i o u]
+    arr_str = str.split
+  
+    arr_str.map! do |word|
+      if vowels.include?(word[0])
+        word + "ay"
+      else
+        if word.start_with?("qu")
+          prefix = "qu"
+          rest = word[2..]
+          word = rest + prefix + "ay"
+        elsif word.include?("qu")
```

Comment:
There's a bug with words that contain `qu` after a vowel. For example, `translate("sequel")` is `elsequay` with your implementation, but it should be `equelsay`. However, because this kind of word isn't in the test cases, fixing this bug isn't necessary to pass the assignment.